### PR TITLE
Flare Buff

### PR DESCRIPTION
**Pull request by Geevies**
* Flares now burn brighter, longer, are smaller items (can bring more), and flare boxes now contain 12 of them, up from 4.

### CODE REVIEW

**Review by hazelrat — APPROVED** [review]

**Comment by alsoandanswer** [comment]
This seems like a bit much for a cheap, disposable item, wouldn't it be better to have better flashlights and floodlights to set up instead?

**Comment by hazelrat** [comment]
> This seems like a bit much for a cheap, disposable item, wouldn't it be better to have better flashlights and floodlights to set up instead?

I think we want flares to be decently strong for how important lighting is on dark odyssey maps like the Cryo Outpost. The more people can see the better, and handling flare lighting gives support roles on the expedition (service, operations, etc) something to do.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
They should be a different item entirely, then, like an "expeditionary flare" or something. These are balanced around the Horizon's corridors, rooms and spaces, compared to the expanses of an exoplanet.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -1211,12 +1211,12 @@
 	name = "flares box"
 	desc = "A box full of flares."
 	foldable = FALSE
-	max_storage_space = 4
+	max_storage_space = 12
```
Comment (alsoandanswer) [inline code/game/objects/items/weapons/storage/boxes.dm:1214]: You should make a new "large box" variant instead, e.g. the large produce box used in the kitchen.

File: code/game/objects/items/devices/lighting/flare.dm [hunk 2]
```diff
@@ -2,9 +2,9 @@
 	name = "flare"
 	desc = "A red standard-issue flare. There are instructions on the side reading 'twist cap off, make light'."
 	desc_info = "Use this item in your hand, to turn on the light."
-	w_class = WEIGHT_CLASS_SMALL
-	brightness_on = 3 // Pretty bright.
-	light_power = 4
+	w_class = WEIGHT_CLASS_TINY
```
Comment (alsoandanswer) [inline code/game/objects/items/devices/lighting/flare.dm:5]: Don't buff the old flares, they're used for shitty emergency kits.

Make a new variant.

**Review by hazelrat — APPROVED** [review]
Same as Dreamy, I don't think I see flares getting enough usage as is to be convinced this would be a significant balance issue.

**Comment by OolongCow** [comment]
The only time you would ever willingly bring flares at the moment is if you just don't have access to other options, because flares are just bad at their niche of being able to cover lots of tiles with temporary light. They are incredibly space-inefficient for no real reason and in the vast majority of cases there is zero advantage to not just drag floodlights with you. That is not a "nerf floodlights!" demand, but proof that what flares are SUPPOSED to be good for, they just aren't.

**Pull request merged**